{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "8d640521-9227-42e8-be71-3a194d63ac87",
   "metadata": {},
   "outputs": [],
   "source": [
    "import site\n",
    "site.addsitedir(\"/rhome/kzhao028/.local/lib/python3.11/site-packages\")\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import os\n",
    "import glob\n",
    "import gffutils\n",
    "\n",
    "import seaborn as sns\n",
    "import scipy as scipy\n",
    "import sys\n",
    "import matplotlib\n",
    "matplotlib.style.use('ggplot')\n",
    "idx = pd.IndexSlice\n",
    "\n",
    "from IPython.core.interactiveshell import InteractiveShell\n",
    "InteractiveShell.ast_node_interactivity = \"all\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "7b08d830-6213-4823-ad13-069128e6cdd3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_2_mCh_rep2.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "fqlib=glob.glob('/rhome/kzhao028/shared/seqdata/20251202*/*fastq.gz')\n",
    "print(fqlib[0])\n",
    "bamlist=glob.glob('./bamfiles_hg38/*bam')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "37936979-f504-4740-8d9b-ff53ff4aa4a1",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Samples to process: 15\n",
      "Existing BAMs: 0\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_1_mCh_rep1.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_1_mCh_rep1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_5_PB1_rep2.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_5_PB1_rep2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_2_mCh_rep2.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_2_mCh_rep2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_10_PB1+2_rep1.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_10_PB1+2_rep1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_14_All_rep2.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_14_All_rep2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_3_mCh_rep3.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_3_mCh_rep3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_13_All_rep1.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_13_All_rep1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_9_PB2_rep3.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_9_PB2_rep3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_15_All_rep3.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_15_All_rep3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_11_PB1+2_rep2.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_11_PB1+2_rep2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_6_PB1_rep3.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_6_PB1_rep3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_12_PB1+2_rep3.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_12_PB1+2_rep3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_4_PB1_rep1.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_4_PB1_rep1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_8_PB2_rep2.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_8_PB2_rep2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n",
      "STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 --readFilesIn /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/D25XYN_7_PB2_rep1.fastq.gz --readFilesCommand zcat --outFileNamePrefix /rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/D25XYN_7_PB2_rep1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "\n",
    "samplename_list = []\n",
    "sname_list = []\n",
    "\n",
    "def strip_fastq_ext(fname):\n",
    "    \"\"\"Remove .fastq.gz or .fq.gz but keep entire basename.\"\"\"\n",
    "    if fname.endswith(\".fastq.gz\"):\n",
    "        return fname[:-9]\n",
    "    if fname.endswith(\".fq.gz\"):\n",
    "        return fname[:-6]\n",
    "    return os.path.splitext(fname)[0]\n",
    "\n",
    "# Build sample lists\n",
    "for l in fqlib:\n",
    "    if 'EKDL' not in l:\n",
    "        base = os.path.basename(l)\n",
    "        name = strip_fastq_ext(base)\n",
    "        samplename_list.append(name)\n",
    "        sname_list.append(name)\n",
    "\n",
    "# Build list of existing BAM files to skip\n",
    "bname_list = []\n",
    "for b in bamlist:\n",
    "    if 'EKDL25' not in b:\n",
    "        base = os.path.basename(b)\n",
    "        name = strip_fastq_ext(base)\n",
    "        bname_list.append(name)\n",
    "\n",
    "# Identify samples that still need STAR alignment\n",
    "the_rest_list = list(set(sname_list).difference(bname_list))\n",
    "print(\"Samples to process:\", len(the_rest_list))\n",
    "print(\"Existing BAMs:\", len(bname_list))\n",
    "\n",
    "# Output directory\n",
    "outdir = \"/rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/star_output/\"\n",
    "indir  = \"/rhome/kzhao028/shared/seqdata/20251202-Neil_Flu_RNAseq/\"\n",
    "\n",
    "# Build STAR commands for single-end reads\n",
    "cmdlist = []\n",
    "for sample in the_rest_list:\n",
    "\n",
    "    fastq_path = os.path.join(indir, sample + \".fastq.gz\")\n",
    "    prefix = os.path.join(outdir, sample + \"_\")\n",
    "\n",
    "    starcmd = (\n",
    "        'STAR --genomeDir /rhome/kzhao028/shared/Joy/references/human_flu/fluWSN_nicole_hg38v47 '\n",
    "        '--readFilesIn ' + fastq_path +\n",
    "        ' --readFilesCommand zcat '\n",
    "        '--outFileNamePrefix ' + prefix +\n",
    "        ' --outSAMtype BAM SortedByCoordinate --runThreadN 16\\n'\n",
    "    )\n",
    "     #   'mv ' + prefix + '*out* '\n",
    "      #  '/rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\\n'\n",
    "       # )\n",
    "\n",
    "\n",
    "    print(starcmd)\n",
    "    cmdlist.append(starcmd)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "6475b0ec-c26e-4f5d-b8ba-9ae94064e40a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "15"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "18"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "27"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "19"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "26"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "25"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "19"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "17"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "21"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr87_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr87_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr61_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr61_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_GFP_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_GFP_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "449"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr58_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_cr58_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "445"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr87_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr87_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr61_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr61_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_PBS_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_PBS_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_PBS_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_PBS_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "449"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_PBS_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_PBS_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_PBS_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_PBS_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "441"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_GFP_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_GFP_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "441"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr58_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_cr58_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr87_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_cr87_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr87_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr87_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_GFP_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_GFP_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "449"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr87_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_cr87_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "445"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr61_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_cr61_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "445"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_PBS_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_PBS_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_PBS_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_PBS_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "449"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr87_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_cr87_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr87_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_cr87_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr58_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr58_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_PBS_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_PBS_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_PBS_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_PBS_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "441"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr58_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_cr58_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_PBS_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_PBS_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_PBS_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_PBS_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "441"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_PBS_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut1804_PBS_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_PBS_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut1804_PBS_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "449"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr58_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv mut5646_cr58_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "453"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr87_r3_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_cr87_r3_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "445"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr58_r2_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_cr58_r2_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "445"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr61_r1_1.fq.gz /rhome/jshengna/shared/seqdata/20250407-IRD_RNA_editing/IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 16\n",
      "mv IRD22_cr61_r1_*out* /rhome/jshengna/shared/Joy/20250419-IRD_rna_editing/star_output/\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "445"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f=open('run_the_rest_batch.sh','w')\n",
    "\n",
    "#!/bin/bash -l\n",
    "\n",
    "#SBATCH --nodes=1\n",
    "#SBATCH --ntasks=1\n",
    "#SBATCH --cpus-per-task=10\n",
    "#SBATCH --mem=10G\n",
    "#SBATCH --time=1-00:15:00     # 1 day and 15 minutes\n",
    "#SBATCH --mail-user=useremail@address.com\n",
    "#SBATCH --mail-type=ALL\n",
    "#SBATCH --job-name=\"just_a_test\"\n",
    "#SBATCH -p epyc # You can use any of the following; epyc, intel, batch, highmem, gpu\n",
    "\n",
    "\n",
    "f.write('#!/bin/bash -l\\n')\n",
    "f.write('#SBATCH --nodes=1\\n')\n",
    "f.write('#SBATCH --cpus-per-task=16\\n')\n",
    "f.write('#SBATCH --mem=128G\\n')\n",
    "f.write('#SBATCH --time=2-00:00:00\\n')\n",
    "f.write('#SBATCH --job-name=\"ird\"\\n')\n",
    "f.write('#SBATCH -p highmem\\n')\n",
    "f.write('module load star\\n')\n",
    "f.write('module load samtools\\n')\n",
    "\n",
    "for c in cmdlist:\n",
    "    print(c)\n",
    "    f.write(c)\n",
    "\n",
    "f.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "c53ac533-411a-4559-885b-e9cf6359c034",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "19"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bamlist=glob.glob('./bamfiles_hg38/*bam')\n",
    "len(bamlist)\n",
    "updated_cmdlist=[]\n",
    "for i,c in enumerate(cmdlist):\n",
    "    for b in bamlist:\n",
    "        bname='_'.join(os.path.basename(b).split('_')[:-1])\n",
    "        # print(bname)\n",
    "        if bname not in c:\n",
    "            updated_cmdlist.append(c)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "75e446c8-27fb-4283-bd77-49330fb072e8",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b5810b11-9574-4763-8dc6-cd14f48970bd",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "72"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "19"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(cmdlist)\n",
    "updated_cmdlist=cmdlist\n",
    "\n",
    "bamlist=glob.glob('./bamfiles_hg38/*bam')\n",
    "len(bamlist)\n",
    "# print(bamlist)\n",
    "\n",
    "for b in bamlist:\n",
    "    bname='_'.join(os.path.basename(b).split('_')[:-1])\n",
    "    # print(bname)\n",
    "    for c in cmdlist:\n",
    "        if bname not in c:\n",
    "            updated_cmdlist.append(c)\n",
    "f.close()\n",
    "\n",
    "print(updated_cmdlist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "807e6b78-ea54-4f4c-bec0-590431a5dbbb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "17"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "text/plain": [
       "21"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r2_1.fq.gz mut5646_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r1_1.fq.gz mut5646_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r1_1.fq.gz IRD22_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r1_1.fq.gz mut5646_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r2_1.fq.gz mut1804_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r2_1.fq.gz mut1804_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r1_1.fq.gz IRD22_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr61_r3_1.fq.gz mut5646_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r2_1.fq.gz mut5646_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r2_1.fq.gz IRD22_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r2_1.fq.gz mut5646_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r3_1.fq.gz IRD22_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r1_1.fq.gz mut1804_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r1_1.fq.gz mut1804_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r3_1.fq.gz mut5646_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r1_1.fq.gz IRD22_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r1_1.fq.gz mut1804_cr61_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r1_1.fq.gz mut5646_GFP_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_GFP_r2_1.fq.gz IRD22_GFP_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_GFP_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr87_r1_1.fq.gz mut5646_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r3_1.fq.gz IRD22_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r2_1.fq.gz mut5646_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r2_1.fq.gz IRD22_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r3_1.fq.gz mut1804_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r3_1.fq.gz IRD22_cr87_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r3_1.fq.gz mut1804_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr61_r2_1.fq.gz IRD22_cr61_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr61_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r1_1.fq.gz mut1804_cr58_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr58_r3_1.fq.gz IRD22_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_GFP_r3_1.fq.gz mut5646_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut5646_cr58_r3_1.fq.gz mut5646_cr58_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut5646_cr58_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr87_r2_1.fq.gz mut1804_cr87_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr87_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn IRD22_cr87_r1_1.fq.gz IRD22_cr87_r1_2.fq.gz --readFilesCommand zcat --outFileNamePrefix IRD22_cr87_r1_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "244"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_GFP_r3_1.fq.gz mut1804_GFP_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_GFP_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "247"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr61_r3_1.fq.gz mut1804_cr61_r3_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr61_r3_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "STAR --genomeDir /rhome/jshengna/shared/Joy/references/hg38/hg38_STAR_index --readFilesIn mut1804_cr58_r2_1.fq.gz mut1804_cr58_r2_2.fq.gz --readFilesCommand zcat --outFileNamePrefix mut1804_cr58_r2_ --outSAMtype BAM SortedByCoordinate --runThreadN 6\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "250"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f=open('run_the_rest.sh','w')\n",
    "f.write('module load star\\n')\n",
    "f.write('module load samtools\\n')\n",
    "for c in updated_cmdlist:\n",
    "    print(c)\n",
    "    f.write(c)\n",
    "\n",
    "f.close()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5a99ac50-b046-4ea9-acb6-620e04d5adfc",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
